tests/test_install.py

```diff
@@ -377,3 +377,61 @@ def test_install_fetch_missing_python_invalid(capsys, python_version):
     assert run_pipx_cli(["install", "--python", python_version, "--fetch-missing-python", "pycowsay"])
     captured = capsys.readouterr()
     assert f"No executable for the provided Python version '{python_version}' found" in captured.out
+
+
+def test_install_run_in_separate_directory(caplog, capsys, pipx_temp_env, monkeypatch, tmp_path):
```

Comment:
Could you explain how this change is related to the feature introduced?

Reply:
Sorry, I have to admit this test case is removed unexpectedly by https://github.com/pypa/pipx/pull/1350.

I did not realize that until I encounter conflicts when rebasing from main branh. :-(

Reply:
I guess it's all right, then, thanks for clarifying.